apps/newsletters-ui/src/app/components/NewsletterDetails.tsx

```diff
@@ -6,19 +6,22 @@ import {
 	textSansObjectStyles,
 } from '@guardian/source-foundations';
 import type { Newsletter } from '@newsletters-nx/newsletters-data-client';
-import { getPalette } from '../util';
+import {
+	getPropertyDescription,
+	isPropertyOptional,
+} from '@newsletters-nx/newsletters-data-client';
+import { getGuardianUrl, getPalette } from '../util';
 import type { SourcePalette } from '../util';
+import { Illustration } from './Illustration';
 
 interface Props {
 	newsletter: Newsletter;
 }
 
-const detailStyles = (palette: SourcePalette) => css`
-	h2 {
-		${headlineObjectStyles.medium()};
-		color: ${palette[100]};
-		border-bottom: 2px solid ${palette[400]};
-		margin: 0 0 ${space[3]}px 0;
+const tableStyles = (palette: SourcePalette) => css`
```

Comment:
Can do - I don't have a preference. 
Looking at this file again, it would probably make sense to break out a component that renders the tables with the fields in. That would give us two smaller files and make the refactoring and/or redesigning easier